sources/lib/api/gcp/context/_utils.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2016 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Support for getting gcloud credentials. """
+
+import argparse
+import oauth2client.client
+import oauth2client.file
+import oauth2client.tools
+import json
+import os
+
+
+def get_config_dir():
+  config_dir = os.getenv('CLOUDSDK_CONFIG')
```

Comment:
I'm not sure what you mean here. We are using the same environment variables that gcloud uses; that's useful again for people who want to run the CLI commands in Datalab.